prow-jobs/pingcap-tiflow-latest-presubmits.yaml

```diff
@@ -39,6 +39,18 @@ presubmits:
       rerun_command: "/test cdc-integration-storage-test"
       branches:
         - ^master$
+    - name: pingcap/tiflow/pull_cdc_integration_pulsar_test
+      agent: jenkins
+      decorate: false # need add this.
+      always_run: false
+      skip_if_only_changed: "(\\.md|Dockerfile|OWNERS|OWNERS_ALIASES)$"
```

Comment:
```suggestion
```
please remove this line to avoid being triggered by other PRs during the testing period.